### DISCUSSION TITLE
Default Sharepoint bar + parse search into the search result

### DISCUSSION THREAD

**Discussion by tocri** [microsoft-search/pnp-modern-search#1848] [Q&A] [answered]
Hi, i have a little question.

I redirect the default sharepoint search to a result page like you showed .
Now if i write a term on the sharepoint search, the term in not parsed to the webpart. 
BTW, the redirection of the page works. 

What am i missing ?

Thanks

**Comment by FranckyC** [comment, marked as answer]
Hi @tocri, in the Search Box settings on the target page, you must use the **Use dynamic data source as default input** setting and configure it to either _Page environment->Search query_ or _Page environment->QueryString_
